src/Elements/Geometry/Transform.cs

```diff
@@ -225,7 +225,8 @@ public Line OfLine(Line line)
         /// <returns>A new plane transformed by this transform.</returns>
         public Plane OfPlane(Plane plane)
         {
-            return new Plane(OfPoint(plane.Origin), OfVector(plane.Normal));
+            var tempXform = new Transform(Vector3.Origin, this.XAxis, this.ZAxis);
```

Comment:
My understanding of what this fix does is that, by creating a separate transform with zero translation, and transforming the initial plane's vector by that, we don't accidentally accumulate the translate of the the transform onto the vector. Is that correct? If so, nice!

Reply:
Exactly! I'll leave a comment to that effect.